### PR TITLE
Update citus memory params for new machine

### PR DESCRIPTION
**Pull request by calellowitz**
##### SUMMARY

There was a lot of contention the machine in the shared buffers so I took the opportunity to update these for the new specs. Kept shared buffers at 25% made effective cache size 50% rather than the 75% before because it looks like the machine had memory pressure, but happy to change that one.

### CODE REVIEW

File: environments/icds-cas/inventory/citusdb.csv [hunk 1]
```diff
@@ -41,13 +41,13 @@ citusdb_master,postgresql_max_wal_size,8GB,S,,,,,,
...
 citusdb_master,postgresql_random_page_cost,1.1,F,,,,,,
-citusdb_master,postgresql_shared_buffers,20GB,S,,,,,,
+citusdb_master,postgresql_shared_buffers,40GB,S,,,,,,
 citusdb_master,postgresql_wal_buffers,16MB,S,,,,,,
 citusdb_master,postgresql_work_mem,1872kB,S,,,,,,
 citusdb_worker,pgbouncer_default_pool,450,I,,,,,,
 citusdb_worker,postgresql_checkpoint_completion_target,0.9,F,,,,,,
 citusdb_worker,postgresql_default_statistics_target,500,I,,,,,,
-citusdb_worker,postgresql_effective_cache_size,60GB,S,,,,,,
+citusdb_worker,postgresql_effective_cache_size,80GB,S,,,,,,
```
Comment (emord) [inline environments/icds-cas/inventory/citusdb.csv]: I think this should have been on `citusdb_master`

Comment (calellowitz) [reply environments/icds-cas/inventory/citusdb.csv]: Yep it should be

Comment (calellowitz) [reply environments/icds-cas/inventory/citusdb.csv]: Fixed

**Comment by emord** [comment]
I verified these are the settings that are live on citusmaster1

**Pull request merged**